src/components/pages/Main/RecordsBody.tsx

```
@@ -0,0 +1,65 @@
+'use client';
+
+import { useEffect, useRef } from 'react';
+import { ChatCard, CHAT_CARD_COLOR, CHAT_CARD_STATUS, Header, HEADER_VARIANT } from '@/components';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)